packages/test-in-console/run.sh

```diff
@@ -10,6 +10,10 @@ export METEOR_HOME=`pwd`
 
 export phantom=$phantom
 
+# force fibers to be disabled
+export DISABLE_FIBERS=1
+export IGNORE_ASYNC_PLUGIN=1
```

Comment:
These env vars should no longer be needed. If they are, someplace is using an old version of meteor-babel.